frontend/src/metabase/auth/containers/AuthLayout/AuthLayout.tsx

```diff
@@ -3,7 +3,8 @@ import AuthLayout from "../../components/AuthLayout";
 import { State } from "metabase-types/store";
 
 const mapStateToProps = (state: State) => ({
-  showIllustration: state.settings.values["show-lighthouse-illustration"],
+  showIllustration:
+    state.settings.values["show-lighthouse-illustration"] ?? true,
```

Comment:
I know this is just a patch that'll be reverted in 0.45, but do you think we should forbid accessing `state.settings` directly and rely on [`settings.ts`](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/lib/settings.ts) instead?

I feel like having multiple ways of doing the same thing will create more confusion. What do you think?

Reply:
I'm not sure. Using `state.settings` works when the settings are changed or reloaded and is far easier to test. Maybe we should actually deprecate `Settings` class and rely on the redux store instead.

Reply:
This is interesting. I wonder how it'll pan out.